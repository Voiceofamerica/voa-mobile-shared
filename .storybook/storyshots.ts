
import initStoryshots from '@storybook/addon-storyshots'

jest.mock('react-dom', () => {
  return {
    render: () => null,
    unmountComponentAtNode: () => null,
    findDOMNode: () => { return {} },
  }
})

initStoryshots()
