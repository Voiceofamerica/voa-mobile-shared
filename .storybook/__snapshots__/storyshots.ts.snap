// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots BottomNav with active item 1`] = `
<div
  className="bottomNav "
  style={undefined}
>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    AB
  </button>
  <button
    className="iconItem  activeItem"
    onClick={[Function]}
  >
    CD
  </button>
  <button
    className="roundItem "
    onClick={[Function]}
    style={undefined}
  >
    EF
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    GH
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    IJ
  </button>
</div>
`;

exports[`Storyshots BottomNav with active round item 1`] = `
<div
  className="bottomNav "
  style={undefined}
>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    AB
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    CD
  </button>
  <button
    className="roundItem activeItem "
    onClick={[Function]}
    style={undefined}
  >
    EF
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    GH
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    IJ
  </button>
</div>
`;

exports[`Storyshots BottomNav with round item 1`] = `
<div
  className="bottomNav "
  style={undefined}
>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    AB
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    CD
  </button>
  <button
    className="roundItem "
    onClick={[Function]}
    style={undefined}
  >
    EF
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    GH
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    IJ
  </button>
</div>
`;

exports[`Storyshots BottomNav without round item 1`] = `
<div
  className="bottomNav "
  style={undefined}
>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    AB
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    CD
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    EF
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    GH
  </button>
  <button
    className="iconItem "
    onClick={[Function]}
  >
    IJ
  </button>
</div>
`;

exports[`Storyshots Card display 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="gradient"
    >
      <div
        className="titleText"
      >
        <span>
           
          President Trump: From North Korea to Iran, we confront dangerous governments
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Card display with icon 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="gradient"
    >
      <div
        className="titleText"
      >
        <span>
          <img
            src="Some File"
          />
           
          President Trump: From North Korea to Iran, we confront dangerous governments
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots MediaPlayer display video 1`] = `
<video
  autoPlay={undefined}
  className=""
  controls={true}
  onPause={[Function]}
  onPlay={[Function]}
  playsInline={true}
  src="https://av.voanews.com/Videoroot/Pangeavideo/2017/12/1/14/146a3e25-ebcf-444e-b1dd-133cf64a46e4_mobile.mp4"
  style={undefined}
/>
`;

exports[`Storyshots MediaPlayer play audio 1`] = `
<video
  autoPlay={undefined}
  className=""
  controls={true}
  onPause={[Function]}
  onPlay={[Function]}
  playsInline={true}
  src="https://av.voanews.com/clips/VCH/2017/12/06/39923226-d756-47fd-bdf4-b5b6523d9c1f.mp3"
  style={undefined}
/>
`;

exports[`Storyshots ResilientImage display 1`] = `
<div
  className="resilientImage "
  onClick={[Function]}
  style={undefined}
>
  <img
    className="imageContent"
    src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
  />
</div>
`;

exports[`Storyshots ResilientImage display without image 1`] = `
<div
  className="resilientImage "
  onClick={[Function]}
  style={undefined}
>
  <img
    className="imageContent"
    src="asdf"
  />
</div>
`;

exports[`Storyshots SecondaryCard display two 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="titleText"
    >
      President Trump: From North Korea to Iran, we confront dangerous governments
    </div>
  </div>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/01D5530E-3642-4CBF-957E-AE990ABE9167_cx0_cy14_cw0_w650_r1.jpg"
      />
    </div>
    <div
      className="titleText"
    >
      At least seven people were killed in a police vehicle in Balochistan, Pakistan
    </div>
  </div>
</div>
`;

exports[`Storyshots SecondaryCard rtl display two 1`] = `
<div
  dir="rtl"
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="titleText"
    >
      President Trump: From North Korea to Iran, we confront dangerous governments
    </div>
  </div>
  <div
    className="card undefined"
    onClick={[Function]}
    style={undefined}
  >
    <div
      className="resilientImage imageContainer"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/01D5530E-3642-4CBF-957E-AE990ABE9167_cx0_cy14_cw0_w650_r1.jpg"
      />
    </div>
    <div
      className="titleText"
    >
      At least seven people were killed in a police vehicle in Balochistan, Pakistan
    </div>
  </div>
</div>
`;

exports[`Storyshots Spinner display 1`] = `
<svg
  className="spinner "
  style={undefined}
  viewBox="0 0 100 100"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    className="path"
    cx={50}
    cy={50}
    fill="none"
    r={45}
    strokeLinecap="round"
    strokeWidth={10}
  />
</svg>
`;

exports[`Storyshots SvgIcon display 1`] = `
<div
  className="icon "
  style={
    Object {
      "color": "#53A68E",
      "fontSize": 100,
      "webkitMaskImage": "url(Some File)",
    }
  }
/>
`;

exports[`Storyshots SwipeToDelete display 1`] = `
<div
  className="swipeToDelete animated"
  onMouseDown={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "opacity": 1,
      "right": 0,
    }
  }
>
  <div
    style={
      Object {
        "background": "#888",
      }
    }
  >
    Hello
  </div>
</div>
`;

exports[`Storyshots Switch display off 1`] = `
<div
  className="innerSwitch "
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="toggle"
  />
</div>
`;

exports[`Storyshots Switch display on 1`] = `
<div
  className="innerSwitch on "
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="toggle"
  />
</div>
`;

exports[`Storyshots Switch interactible 1`] = `
<div
  className="innerSwitch "
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="toggle"
  />
</div>
`;

exports[`Storyshots Ticket Display display 1`] = `
<div
  className="ticketDisplay "
  style={undefined}
>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Ticket Display rtl display 1`] = `
<div
  dir="rtl"
>
  <div
    className="ticketDisplay "
    style={undefined}
  >
    <div
      className="ticket"
      onClick={[Function]}
    >
      <div
        className="resilientImage imageArea show"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="imageContent"
          src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
        />
      </div>
      <div
        className="content"
      >
        <div
          className="textContent"
        >
          <div
            className="titleText"
          >
            President Trump: From North Korea to Iran, we confront dangerous governments
          </div>
        </div>
        <div
          className="minorText"
        >
          15:08
        </div>
      </div>
    </div>
    <div
      className="ticket"
      onClick={[Function]}
    >
      <div
        className="resilientImage imageArea show"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="imageContent"
          src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
        />
      </div>
      <div
        className="content"
      >
        <div
          className="textContent"
        >
          <div
            className="titleText"
          >
            President Trump: From North Korea to Iran, we confront dangerous governments
          </div>
        </div>
        <div
          className="minorText"
        >
          15:08
        </div>
      </div>
    </div>
    <div
      className="ticket"
      onClick={[Function]}
    >
      <div
        className="resilientImage imageArea show"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="imageContent"
          src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
        />
      </div>
      <div
        className="content"
      >
        <div
          className="textContent"
        >
          <div
            className="titleText"
          >
            President Trump: From North Korea to Iran, we confront dangerous governments
          </div>
        </div>
        <div
          className="minorText"
        >
          15:08
        </div>
      </div>
    </div>
    <div
      className="ticket"
      onClick={[Function]}
    >
      <div
        className="resilientImage imageArea show"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="imageContent"
          src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
        />
      </div>
      <div
        className="content"
      >
        <div
          className="textContent"
        >
          <div
            className="titleText"
          >
            President Trump: From North Korea to Iran, we confront dangerous governments
          </div>
        </div>
        <div
          className="minorText"
        >
          15:08
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Ticket display 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Ticket display with icon 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
      <div
        className="childrenArea"
      >
        <img
          src="Some File"
        />
      </div>
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Ticket rtl display 1`] = `
<div
  dir="rtl"
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Ticket rtl display with icon 1`] = `
<div
  dir="rtl"
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div
    className="ticket"
    onClick={[Function]}
  >
    <div
      className="resilientImage imageArea show"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="imageContent"
        src="https://gdb.voanews.com/3E1C440D-1C78-4B88-84F7-ECB12FB5B874_cx3_cy0_cw97_w650_r1.png"
      />
      <div
        className="childrenArea"
      >
        <img
          src="Some File"
        />
      </div>
    </div>
    <div
      className="content"
    >
      <div
        className="textContent"
      >
        <div
          className="titleText"
        >
          President Trump: From North Korea to Iran, we confront dangerous governments
        </div>
      </div>
      <div
        className="minorText"
      >
        15:08
      </div>
    </div>
  </div>
</div>
`;
